core/src/main/java/org/ehcache/core/internal/util/Pacer.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.core.internal.util;
+
+import org.ehcache.core.spi.time.TimeSource;
+
+/**
+ * Class used to pace a call to prevent it from happening too frequently.
+ */
+public class Pacer {
+
+  private volatile long nextLogTime = 0;
+  private final TimeSource timeSource;
+  private final long delay;
+
+  /**
+   * Unique constructor
+   *
+   * @param delay delay between each call paced
+   */
+  public Pacer(TimeSource timeSource, long delay) {
```

Comment:
Should we be 310-ing it up in here?

Reply:
Since it is internal stuff, I don't think it's needed